src/test/static/analysis/namespaces/analysis.json

```diff
@@ -0,0 +1,94 @@
+{
+  "namespaces": [
+    {
+      "description": "This is the \"aaa\" namespace.",
+      "name": "aaa",
+      "sourceRange": {
+        "end": {
+          "column": 4,
+          "line": 31
+        },
+        "file": "namespaces.html",
+        "start": {
+          "column": 2,
+          "line": 7
+        }
+      },
+      "summary": "This is the \"aaa\" namespace. (summary)",
+      "functions": [
+        {
+          "description": "This is a simple namespace method.",
+          "name": "aaa.sum",
```

Comment:
Names include the namespace. We may want to change this in the (near) future, but that's how it is now.